javadoc/help-doc.html

```diff
@@ -26,25 +17,20 @@
     catch(err) {
     }
 //-->
-var pathtoroot = "./";
-var useModuleDirectories = true;
-loadScripts(document, 'script');</script>
+</script>
 <noscript>
 <div>JavaScript is disabled on your browser.</div>
 </noscript>
-<header role="banner">
-<nav role="navigation">
-<div class="fixedNav">
 <!-- ========= START OF TOP NAVBAR ======= -->
-<div class="topNav"><a id="navbar.top">
+<div class="topNav"><a name="navbar.top">
```

Comment:
seems like there are some changes to HTMLs not related to the version change. Have you tested out if the UI still displays correctly?

Reply:
I opened up the index.html and clicked around to validate there were no issues. I opened up the help-doc (this page) and a few other class specific docs, and other pages and don't see any issues. Let me know what you think